lib/internal/Magento/Framework/Mail/Template/TransportBuilder.php

```diff
@@ -75,14 +75,14 @@ class TransportBuilder
 
     /**
      * @param FactoryInterface $templateFactory
-     * @param \Magento\Framework\Mail\Message $message
+     * @param \Magento\Framework\Mail\MessageInterface $message
      * @param \Magento\Framework\Mail\Template\SenderResolverInterface $senderResolver
      * @param \Magento\Framework\ObjectManagerInterface $objectManager
      * @param \Magento\Framework\Mail\TransportInterfaceFactory $mailTransportFactory
      */
     public function __construct(
         \Magento\Framework\Mail\Template\FactoryInterface $templateFactory,
-        \Magento\Framework\Mail\Message $message,
+        \Magento\Framework\Mail\MessageInterface $message,
```

Comment:
Message is non-injectable. Shouldn't be constructor argument. But it wasn't added in this PR